src/test/java/fi/metatavu/metamind/functional/resources/MysqlResource.java

```diff
@@ -0,0 +1,48 @@
+package fi.metatavu.metamind.functional.resources;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.testcontainers.containers.MySQLContainer;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class MysqlResource implements QuarkusTestResourceLifecycleManager {
+      private static final String DATABASE = "db";
+      private static final String USERNAME = "user";
+      private static final String PASSWORD = "pass";
+
+    static final MySQLContainer db = new SpecifiedMySQLContainer("mysql:5.6")
+        .withDatabaseName(DATABASE)
+        .withUsername(USERNAME)
+        .withPassword(PASSWORD)
+        .withCommand(
+            "--character-set-server=utf8mb4",
+                    "--collation-server=utf8mb4_unicode_ci",
+                    "--lower_case_table_names=1"
+    );
+
+    @Override
+    public Map<String, String> start() {
+        db.start();
+
+        HashMap config = new HashMap<String, String>();
+        config.put("quarkus.datasource.username", USERNAME);
+        config.put("quarkus.datasource.password", PASSWORD);
+        config.put("quarkus.datasource.jdbc.url", db.getJdbcUrl());
+        return config;
+    }
+
+    @Override
+    public void stop() {
+        db.stop();
+    }
+
+}
+
+class SpecifiedMySQLContainer extends MySQLContainer<SpecifiedMySQLContainer>
+{
+    public SpecifiedMySQLContainer(String image){
+        super(image);
+    }
+
+}
```

Comment:
Add docs to class and method
```suggestion
class SpecifiedMySQLContainer extends MySQLContainer<SpecifiedMySQLContainer> {

    public SpecifiedMySQLContainer(String image) {
        super(image);
    }

}
```